### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server Application Pools Edit_Screen Reader: When we are changing the value in "Start up Time out" spin button, Narrator is not reading the changing value name 

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#145]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)
Screen reader
MAS ID Violated: MAS 1.3.1

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Switch on Narrator
4.Tab till Microsoft IIS on top navigation bar and press enter
5.tab till web server and press enter
6.Tab till Application pool and press enter
7.tab till more button(...)and press enter
8.Click on Edit button then navigate to process tab and press enter
9.Tab till Start up Time out spin button and change the value by using up and down arrow key and check narrator is reading that value or not

Actual Result:
When we are changing the value in "Start up Time out" spin button, Narrator is not reading the changing value name 

Expected Result:
When we are changing the value narrator should announce that value

User Impact:
Screen Reader users will face difficulty if narrator will not announce the changing value name

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Reference video:
[Bug_145_Narrator is not reading the changing value.pptx](https://github.com/Microsoft/IIS.WebManager/files/2330859/Bug_145_Narrator.is.not.reading.the.changing.value.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Edge; #Narrator

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
